### PR TITLE
Fix: Change folder structure

### PR DESCRIPTION
**Pull request by jagadeeshi2i**
- move torchserve from custom to v1beta1 -> custom

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by adriangonz** [comment]
Hey @jagadeeshi2i, there seem to be other references to that folder path. E.g. under `./docs/samples/README.md`. Not sure if there are others. 

You may need to find and update those as well.

File: docs/samples/README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ After models are deployed onto model servers with KFServing, you get all the fol
...
 | [Triton Inference Server](https://github.com/triton-inference-server/server) | [TensorFlow,TorchScript,ONNX,TensorRT](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/model_repository.html)| v2 | :heavy_check_mark: | :heavy_check_mark: | [Compatibility Matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html)| [Triton Examples](./v1beta1/triton) |
 | [TFServing](https://www.tensorflow.org/tfx/guide/serving) | [TensorFlow SavedModel](https://www.tensorflow.org/guide/saved_model) | v1 | :heavy_check_mark: | :heavy_check_mark: | [TFServing Versions](https://github.com/tensorflow/serving/releases) | [TensorFlow Examples](./v1alpha2/tensorflow)  |
-| [TorchServe](https://pytorch.org/serve/server.html) | [Eager Model/TorchScript](https://pytorch.org/docs/master/generated/torch.save.html) | v1 | :heavy_check_mark: | WIP | | [TorchServe Examples](./custom/torchserve)  |
+| [TorchServe](https://pytorch.org/serve/server.html) | [Eager Model/TorchScript](https://pytorch.org/docs/master/generated/torch.save.html) | v1 | :heavy_check_mark: | WIP | | [TorchServe Examples](./v1beta1/torchserve)  |
```
Comment (yuzisun) [inline docs/samples/README.md:29]: Also add the reference for torchserve custom example on the custom section? Some description for when need to go custom route would be helpful.

Comment (yuzisun) [reply docs/samples/README.md:29]: Can you also add the supported pytorch version?

**Pull request merged**